src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -543,6 +543,14 @@
 
     public static final Key<Value<Vector3d>> LEFT_ARM_ROTATION = KeyFactory.fake("LEFT_ARM_ROTATION");
 
+    /**
+     * Represents the {@link Key} for representing the dominant {@link HandType}
+     * of a {@link Living} entity.
```

Comment:
I mean, players support hands, so the javadocs should state so. The javadocs should simply state that you cannot **set** a player's primary hand.